### ISSUE TITLE
Update to work with current hcaptcha and talon service

### ISSUE THREAD

**Issue by delebash** [SzymonLisowiec/node-epicgames-client#101]
I have been trying to figure it out myself but I am having trouble.  Currently I am stuck on getting the xal when you need to post to https://talon-service-prod.ecosec.on.epicgames.com/v1/init/execute.  Any idea how I get the xal?  TY.  Very cool app!

**Comment by Revadike** [comment]
Did you figure it out?

https://github.com/Revadike/epicgames-freebies-claimer/pull/184#issuecomment-1040863712

**Comment by delebash** [comment]
No, I did not.  I use a sniffer to grab the token created from the Unreal Launcher, not ideal but works.
